### ISSUE TITLE
After 2019.2.1 hotfix pyrevit 4.7.0 won't start (on Revit 2019)

### ISSUE THREAD

**Issue by MauroSogni** [eirannejad/pyRevit#539]
**Describe the bug**
After installing Revit 2019.2.1 hotfix pyRevit 4.7.0 stopped loading

**Desktop:**
 - OS: WIN 10 Pro
 - pyRevit Version 4.7.0
 - pyRevit Environment: 

```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive)
47beta | Deploy: "basepublic" | Branch: "release/4.7-beta" | Version: "4.7.0-beta" | Path: "C:\Users\mauro.sogni2\AppData\Roaming\pyRevit-47Beta"
==> Attachments
Autodesk Revit 2018.3.2 | Clone: "47beta" | Engine: "277"
==> Installed Extensions
==> Default Extension Search Path
C:\Users\mauro.sogni2\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2018.3.2 | Version: 18.3.2.7 | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018\"
==> Running Revit Instances
==> User Environment
Microsoft Windows 10 [Version 10.0.17763]
Executing User: MAWISA\mauro.sogni2
Active User: MAWISA\mauro.sogni2
Adming Access: No
%APPDATA%: "C:\Users\mauro.sogni2\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
Installed .Net Target Packs: v3.5 v4.0 v4.5 v4.5.1 v4.5.2 v4.6 v4.6.1 v4.6.2 v4.7 v4.7.1 v4.7.2 v4.X
No .Ne-Core Target Packs are installed.
pyRevit CLI 0.12.0.0
```
**pyRevit Loading Error**
[Pyrevit 4.7.0 - Revit 2019.2.1 error.pdf](https://github.com/eirannejad/pyRevit/files/2946638/Pyrevit.4.7.0.-.Revit.2019.2.1.error.pdf)

**Other issues:**
After 2019.2.1 hotfix even pyRevit 4.6.17 does not work it reports a missing pyRevitLoader.dll (do not tested with 2019.2)
By uninstalling 2019.2.1 nothing changes.
Same issue on Revit 2018.3 (but it worked 5 mins ago)

**Comment by eirannejad** [comment]
What is `cpyengine` set to in your pyRevit config file (under `%appdata%/pyRevit`)?

**Comment by eirannejad** [comment]
Okay so as you can see in the report, pyRevit can not see the new Revit 2019 since it does not know its build number. This issue has been resolved for 4.6.18.
